ownCloudAppShared/Client/User Interface/ClientItemCell.swift

```diff
@@ -293,12 +316,18 @@ open class ClientItemCell: ThemeTableViewCell, ItemContainer {
 		}
 	}
 
-	open func titleLabelString(for item: OCItem?) -> String {
```

Comment:
@hosy I think this can be restructured a bit with guards

Reply:
fixed